samples/highcharts/accessibility/accessible-venn/demo.js

```diff
@@ -0,0 +1,283 @@
+        }, {
+            sets: ['5', '2'],
+            value: 3,
+            accessibility: {
+                description: 'The Business Case for Accessibility'
+            },
+            name: 'legal,innovation',
+            color: Highcharts.getOptions().colors[5],
+            dataLabels: {
+                enabled: false
+            }
+        }, {
+            sets: ['2', '4'],
+            value: 1,
+            accessibility: {
+                description: 'The Business Case for Accessibility'
+            },
+            name: 'innovation,reach',
+            color: '#fff',
+            dataLabels: {
+                enabled: false
+            }
+        }, {
+            sets: ['3', '5'],
+            value: 3,
+            accessibility: {
+                description: 'The Business Case for Accessibility'
+            },
+            name: 'brand,legal',
+            color: '#fff',
+            dataLabels: {
+                enabled: false
+            }
+        }, {
+            sets: ['3', '4', '5'],
+            value: 2,
+            accessibility: {
+                description: 'The Business Case for Accessibility'
+            },
+            name: 'brand,reach,legal',
+            color: '#fff',
+            opacity: 1,
+            dataLabels: {
+                enabled: false
+            }
+        }, {
+            sets: ['2', '3', '5'],
+            value: 1,
+            accessibility: {
+                description: 'The Business Case for Accessibility'
+            },
+            name: 'innovation,brand,legal',
+            color: '#fff',
+            opacity: 1,
+            dataLabels: {
+                enabled: false
+            }
+        }, {
+            sets: ['2', '3', '4'],
+            value: 2,
+            accessibility: {
+                description: 'The Business Case for Accessibility'
+            },
+            name: 'innovation,brand,reach',
+            color: Highcharts.getOptions().colors[2],
+            dataLabels: {
+                enabled: false
+            }
+        }, {
+            sets: ['2', '3', '4', '5'],
+            value: 3,
+            accessibility: {
+                description: 'The Business Case for Accessibility'
+            },
+            color: '#fff',
+            opacity: 1,
+            name: 'innovation,brandreachlegal',
+            dataLabels: {
+                enabled: true,
+                useHTML: true,
+                y: -30,
+                x: -30,
+                formatter: function () {
+                    return '<i id="aicon" class="fas fa-universal-access"></i>';
+                }
+            }
+        }]
+    }],
+
+    title: {
+        text: '',
+        style: {
+            fontFamily: 'Roboto',
+            fontSize: '20px',
+            fontWeight: 'bold'
+        }
+    },
+
+    tooltip: {
+        enabled: false
+    },
+
+    exporting: {
+        enabled: false
+    }
+
+});
+
+$('i').mouseover(function (e) {
```

Comment:
Would be nice to get rid of this jQuery :)

Reply:
Agree!